framework/decode/vulkan_ascii_consumer_base.h

```diff
@@ -166,6 +183,21 @@ class VulkanAsciiConsumerBase : public VulkanConsumer, public AnnotationHandler
         fflush(file_);
     }
 
+    /// @brief  Allow callers to be incrementally refactored away from the explicit
+    /// formatting using toStringFlags, tabCount, tabSize.
+    template <typename ToStringFunctionType>
```

Comment:
Later, once all the callers have been updated this will be the signature of the main function above and the thunk through this will go away. I added this now because I want to to use this signature from the callers touched by the core of this change.